python/lsst/ctrl/execute/allocatorParser.py

```
@@ -143,7 +143,7 @@ def parseArgs(self, basename):
             action="store",
             dest="dynamic",
             type=str,
-            default=None,
+            default="__default__",
```

No objections regarding changing the default value, but  the command line help below is slightly misleading. Based on the current description I initially assumed that the `--dynamic`option is a simple on/off flag, but looking at the code I see that one can actually use it to pass a custom template for configuring some aspects of the dynamic slot. So it would be nice if the help was reflecting that, something like:
> a template file to use when configuring dynamic/partitionable slot

---

For the "file input" for --dynamic, I will want to do some  testing as I have not touched that for a while. Plus it ties together with offering memory control in Slurm, so I will do those together at some point, before advertising the feature more loudly.